src/library/scala/collection/mutable/ArrayBuilder.scala

```diff
@@ -34,15 +35,11 @@ sealed abstract class ArrayBuilder[T]
   override def knownSize: Int = size
 
   protected[this] final def ensureSize(size: Int): Unit = {
```

Comment:
it should be called `ensureCapacity`. I've already had coffee, but it made me blink twice. Maybe you think it's too obvious to make the distinction between size and capacity obviouser.

Reply:
in NArr, I'm implementing NArrayBuilder and I definitaly named it `ensureCapacity`.

Isn't it too late to rename it now, though, after 20 years?